var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -130,6 +130,12 @@ class PyNumpy(PythonPackage):
     patch('check_executables4.patch', when='@1.14.0:1.15.4')
     patch('check_executables5.patch', when='@:1.13.3')
 
+    # Backport bug fix for f2py's define for threading when building with Mingw
+    # to 1.20.3:1.22.1 - https://github.com/numpy/numpy/pull/20881/commits
```

Comment:
The second line of this comment can probably be removed, both the URL and versions are in the `patch` command itself.